gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonCallContext.java

```diff
@@ -201,6 +213,10 @@ public HttpJsonCallContext merge(ApiCallContext inputCallContext) {
       newRetryableCodes = this.retryableCodes;
     }
 
+    // The EndpointContext is not updated as there should be no reason for a user
```

Comment:
This a great explanation of `EndpointContext` itself, can you please move it to `EndpointContext` class?

Reply:
Sure, will do.